sdk/compute/Azure.ResourceManager.Compute/api/Azure.ResourceManager.Compute.netstandard2.0.cs

```diff
@@ -4864,6 +4960,12 @@ public partial class ScaleInPolicy
         public static bool operator !=(Azure.ResourceManager.Compute.Models.SecurityEncryptionType left, Azure.ResourceManager.Compute.Models.SecurityEncryptionType right) { throw null; }
         public override string ToString() { throw null; }
     }
+    public partial class SecurityPostureReference
```

Comment:
Could we be more specific on the name of this type?

Reply:
updated to ComputeSecurityPostureReference